web/src/components/wizard/config/ConfigurationStep.tsx

```diff
@@ -141,6 +142,40 @@ const ConfigurationStep: React.FC<ConfigurationStepProps> = ({ onNext }) => {
     return configValues?.[item.name] ?? (item.value || item.default || '');
   };
 
+  // Helper function to determine if default value should be shown
+  const shouldShowDefault = (item: AppConfigItem): boolean => {
+    return !!(item.default && ['text', 'password', 'textarea'].includes(item.type));
+  };
+
+  // Helper function to render help text with default value inline
+  const renderHelpTextWithDefault = (helpText?: string, defaultValue?: string, error?: string) => {
```

Comment:
@JGAntunes was on the fence about this since `default` and `help_text` are two separate properties in the config, but the desire here was to keep them on the same line with each other since they are also very related and both technically "help text". Let me know what you think.